app.js

```diff
@@ -1,5 +1,45 @@
 // import functions and grab DOM elements
-
+// import { compareNumbers } from './utils.js';
 // initialize state
 
-// set event listeners to update state and DOM
\ No newline at end of file
+// set event listeners to update state and DOM
+
+
+const highLow = document.getElementById('highLow');
+let correctNumber = Math.ceil(Math.random() * 10);
+let input = document.getElementById('input');
+// Buttons
+const submit = document.getElementById('submit');
+const reset = document.getElementById('reset');
+
+let guesses = 3;
+
+submit.addEventListener('click', () => {
+    guesses--;
+    if (input.value < correctNumber) {
```

Comment:
you already wrote most of this logic in your compareNumbers function so you'll want to use that here! currently it's only being used in your test